airbyte-integrations/connectors/source-adaptive-insights/unit_tests/test_streams.py

```diff
@@ -107,14 +128,14 @@ def patch_base_class(mocker):
 
 
 def test_request_headers(patch_base_class):
-    stream = AdaptiveInsightsStream(username="a", password="b")
+    stream = AdaptiveInsightsStream(username="a", password="b", version_type="c", start_date="d", accounts="e,f")
     inputs = {}
     expected_headers = {'Content-Type': 'text/xml; charset=UTF-8'}
     assert stream.request_headers(**inputs) == expected_headers
 
 
 def test_parse_response_export_dimensions(patch_base_class, requests_mock):
-    stream = ExportDimensions(username="a", password="b")
+    stream = ExportDimensions(username="a", password="b", version_type="c", start_date="d", accounts="e,f")
```

Comment:
Since you want to validate the same thing for multiple test-cases it's more preferable and readable to use 
https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
Example usage in this [link](https://github.com/Orfium/diaulos/blob/master/tests/test_api/test_run_status_crud.py#L200-L243)